dev/registry.sh

```diff
@@ -192,39 +154,34 @@ elif [ "$cmd" = "create" ]; then
   create_registry
 
 elif [ "$cmd" = "update-manager-local" ]; then
-  build $ROOT/images/manager manager latest
+  build manager latest
 
 # usage: registry.sh update all|dev|api [--include-slim] [--skip-push]
 # if parallel utility is installed, the docker build commands will be parallelized
 elif [ "$cmd" = "update" ]; then
   images_to_build=()
 
   if [ "$sub_cmd" == "all" ]; then
-    cache_builder $ROOT/images/operator operator
-    build_and_push $ROOT/images/operator operator latest
-    images_to_build+=( "${registry_all_images[@]}" )
+    cache_builder operator
+    images_to_build+=( "${non_dev_images[@]}" )
   fi
 
   if [[ "$sub_cmd" == "all" || "$sub_cmd" == "dev" ]]; then
-    cache_builder $ROOT/images/request-monitor request-monitor
-    build_and_push $ROOT/images/request-monitor request-monitor latest
-    images_to_build+=( "${registry_dev_images[@]}" )
+    cache_builder request-monitor
+    images_to_build+=( "${dev_images[@]}" )
   fi
 
-  images_to_build+=( "${registry_base_images[@]}" )
+  images_to_build+=( "${user_facing_images[@]}" )
+
+  if [ "$flag_include_slim" == "true" ]; then
+    images_to_build+=( "${user_facing_slim_images[@]}" )
+  fi
 
   if command -v parallel &> /dev/null ; then
-    images=$(join_by "," "${images_to_build[@]}")
-    flag_include_slim=$flag_include_slim flag_skip_push=$flag_skip_push ecr_logged_in=$ecr_logged_in ROOT=$ROOT REGISTRY_URL=$REGISTRY_URL SHELL=$(type -p /bin/bash) parallel --halt now,fail=1 --eta -d"," --colsep=" " build_and_push "$ROOT/images/{1} {1} latest {2}" ::: "${images[@]}"
+    flag_skip_push=$flag_skip_push ecr_logged_in=$ecr_logged_in ROOT=$ROOT REGISTRY_URL=$REGISTRY_URL SHELL=$(type -p /bin/bash) parallel --will-cite --halt now,fail=1 build_and_push "{} latest" ::: "${images_to_build[@]}"
```

Comment:
nit: same as the one above regarding the `--eta` tag.

Reply:
Yes this was a mistake, thanks!